clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/InfinispanSubsystemXMLReader.java

```diff
@@ -107,7 +107,10 @@ private void parseContainer(XMLExtendedStreamReader reader, PathAddress subsyste
                     break;
                 }
                 case JNDI_NAME: {
-                    readAttribute(reader, i, operation, CacheContainerResourceDefinition.Attribute.JNDI_NAME);
+                    if (this.schema.since(InfinispanSchema.VERSION_5_0)) {
```

Comment:
to help the user migrate from a 4.0 to a 5.0 XML config files, maybe we could log a warning to tell him the jndi-name is deprecated and will be ignored instead of rejecting the config file as a whole. wdyt?

Reply:
It's not in the schema, so it's not valid.  The reason I'm ok with just removing this from the schema altogether, is because it's usage is not supported (nor has it ever been) in EAP.